roles/jenkins-worker/files/etc/docker/daemon.json

```diff
@@ -1,4 +1,5 @@
 {
   "data-root": "/mnt/workdir/containers",
-  "userland-proxy": false
-}
\ No newline at end of file
+  "userland-proxy": false,
+  "registry-mirrors": ["https://mirror.gcr.io"]
```

Comment:
FWIW, they apparently also have limits there.

Reply:
That's.... nice 😁 
Thanks for sharing 🙏 

So now I'm thinking... Should we just set up a mirror similar to what we've done for maven  🤔

Reply:
In theory Nexus supports that, so we could perhaps use our existing instance... though I'm not sure what the limitations are. You'd have to dig a bit :)